base_classes/NXdetector.nxdl.xml

```diff
@@ -889,6 +892,26 @@
        This is the thickness of this converter material.
     </doc>
   </field>
+  <field name="sensor_size" type="NX_FLOAT" units="NX_LENGTH">
```

Comment:
Isn't this recorded in `NXdetector_module`?

Reply:
I do not see any concept `sensor_size` recoded in the `NXdetector_module` base class.

Reply:
@PeterC-DLS we dropped these new `sensor_*` fields from `NXdetector` since they can indeed mostly be described by `NXdetector_module` and there presence in `NXdetector` is not critical for us.